internal/imagedefinition/README.rst

```diff
@@ -108,7 +110,8 @@ The following specification defines what is supported in the YAML:
          # an uncompressed tar archive or a tar archive with one of the
          # following compression types: bzip2, gzip, xz, zstd.
          tarball: (exactly 1 of archive-tasks, seed or tarball must be specified)
-             # The path to the tarball. Can be a local path or an URL.
+             # The path to the tarball. Currently only local paths beginning with
+             # file:// are supported
```

Comment:
Nothing is checking that it's a file:// URI is it?

Reply:
Indeed, maybe we should add `jsonschema:"type=string,format=uri"` to the tarball field and then in validation check.

But I think what Michael has in mind that even for the model assertion field right now, we're only checking if it's an uri, but not actually checking if it's a `file://` uri. Since we're limiting to that, I'd throw some errors if there's something other than file://.

Reply:
I intentionally left out the check for file:// in the hopes that we could add support for retrieving tarballs and model assertions from http(s) sources in the near future